features/view_all_companies.feature

```diff
@@ -131,6 +139,20 @@ Feature: As a visitor,
     And I should not see "5560360793"
     And I should not see t("companies.new_company")
 
+  @selenium @time_adjust
+  Scenario: Visitor sees multiple regions and kommuns for a company in the companies list
+    Given the date is set to "2017-10-01"
+    Given I am Logged out
+    And I am on the "landing" page
+    Then I should see t("companies.index.h_companies_listed_below")
+    And I should see "Company2"
+    And I should see "Västerbotten"
+    And I should see "Norrbotten"
+    And I should see "Uppsala"
+    And I should see "Bromölla"
```

Comment:
Actually, this is a false positive.  This _is_ working in real life (yay!) but this feature is not actually verfiying that it's working. 
"Bromölla" (and perhaps others) is being found in the _search selection list_.

(We have this issue in our other search feature tests I've discovered!)

There are a few ways to address this. Here's the easy, straightforward way, IMNSHO:

- use the same pattern used to count the number of 'things' in the assert_steps.rb file (starting around line 108):  
   1. assign each `td` element a CSS class named 'kommun' (and likewise for region.  Even though we're not using it for CSS styling, it makes it much easier to search and count these specific HTML things.
   2. write a step to search for the number of TDs with the kommun and region CSS classes:
```
Then(/^I should see "([^"]*)" td kommuns/) do |number|
  expect(page).to have_selector('td.kommun', count: number)
end
```

Note that I put "...td..." into the step because it's different from the other existing ones that look for `tr`s right now.  

This will then not pick up and count those Kommuns and Regions in the search list.

Reply:
Wow @weedySeaDragon, good find! 

*makes mental note to not skip negative testing in the future*

I'm not sure I like the solution, because it means putting stuff in production only for testing purposes. But maybe it's the only way.

Reply:
This is known issue: https://github.com/AgileVentures/shf-project/wiki/Work-around-to-Capybara-webdriver-visibility-to-%22select%22-fields

I'd prefer to use the work-around (hide search form) over putting things in production code solely for testing.

Reply:
You're both right -- putting that in there for testing is not a good idea.
I've just changed features for SHF Application where I needed to get the right count; I now have it working by first hiding the search form.  
FWIW: I added a step that just clicks the hide/show button for the search form; I added the step to make it read more naturally.  That will be checked in with my next commits to #459

Reply:
Adding a step that clearly states that it is hiding the search form is a good addition.